doc/dev/developer_guide/index.rst

```diff
@@ -1586,6 +1586,94 @@ server list`` on the teuthology machine, but the target VM hostnames (e.g.
 ``target149202171058.teuthology``) are resolvable within the teuthology
 cluster.
 
+Running Integration Tests using Teuthology
+==========================================
+
+Getting binaries
+----------------
+To run integration tests using teuthology, you need to have Ceph binaries
+built for your branch. Following is how it can be done -
+
+#. Push your branch to `ceph-ci`_ repository. This would trigger the process
+   of building the binaries.
+
+#. To confirm that the build process has initiated, spot your branch name
+   at `shaman`_. Little after the build process has initiated, the single
+   entry with your branch name would multiply, each new entry for a different
+   OS.
+
+#. Wait for building process to complete. Usually, it takes around 2-3 hours
+   depending on how busy the machines used for building are.
+
+Triggering tests
+-----------------
+After building is complete, proceed to trigger tests -
+
+#. Log in to the teuthology machine::
+
+       ssh <username>@teuthology.front.sepia.ceph.com
+
+   You would need access to Sepia lab for this. If you don't have one, you can
+   request that.
+
+#. You would need to get teuthology commands installed. Run the first set of
+   commands in `Running Your First Test`_. After that, activate the virtual
+   environment in which teuthology is installed.
+
+#. Run the ``teuthology-suite`` command::
+
+        teuthology-suite -v -m smithi -c wip-cephfs-shell-feature-x -s fs -p 110 --filter "cephfs-shell"
```

Comment:
> 110

Now this is a delayed run for feature/fix running `< 25` jobs. For example mgr/volumes plugin `--filter volumes` gives only 2 jobs which could be run with a higher priority. Please check https://docs.ceph.com/docs/master/dev/developer_guide/#testing-priority.


Is this an example only for `cephfs-shell`? If not, we could make this command more generic which is appropriate for all users.

Reply:
> > 110
> 
> Now this is a delayed run for feature/fix running `< 25` jobs. For example mgr/volumes plugin `--filter volumes` gives only 2 jobs which could be run with a higher priority. Please check https://docs.ceph.com/docs/master/dev/developer_guide/#testing-priority.
> 


The number present there is just a placeholder.

> Is this an example only for `cephfs-shell`? If not, we could make this command more generic which is appropriate for all users.

There is no such that can be appropriate in all cases to all users.